fl_training.py

```diff
@@ -257,6 +257,7 @@ def compute_test_score(
 
             # Starting model for each node is the aggregated model from the previous mini-batch iteration
             agg_model_for_iteration = [None] * nodes_count
+            aggregation_weights = prepare_aggregation_weights(aggregation_weighting, nodes_count, node_list, local_perf_list)
```

Comment:
We should save the `local_perf_list `in the same way we save `global_val_acc `and `global_val_loss`. This way we would be able to have an insight of how the perf and therefore the weights change/evolve during training.

Reply:
Ok, done